src/main/resources/static/templates/UnrestrictedFileUpload/LEVEL_1/FileUpload.js

```diff
@@ -1,17 +1,14 @@
-function addingEventListenerToUploadImage() {
-  document.getElementById("upload").addEventListener("click", function () {
-    var form = document.getElementById("file");
-    var file = form.files[0];
-    var formData = new FormData();
-    formData.append("file", file);
-    let url = getUrlForVulnerabilityLevel();
-    doPostAjaxCall(uploadImage, url, true, formData);
-  });
-}
-addingEventListenerToUploadImage();
-
 function uploadImage(data) {
-  document.getElementById("uploaded_file_info").innerHTML = data.isValid
-    ? "File uploaded at location:" + data.content
-    : data.content;
```

Comment:
CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')
------
User controlled data in methods like `innerHTML`, `outerHTML` or `document.write` is an anti-pattern that can lead to XSS vulnerabilities